GOVERNANCE.md

```diff
@@ -0,0 +1,132 @@
+# Jujutsu Governance
+
+## Overview
+
+Jujutsu is an open source project, lead, maintained and designed for a worldwide
+community. Anyone who is interested can join, contribute, and participate in the
+decision making process. This document is intended to help you understand how
+you can do that.
+
+## Project roles
+
+There are two broadly defined roles in the Jujutsu project: **Maintainers** and
+**Contributors**.
+
+### Maintainers
+
+At the top of the hierarchy are the **Maintainers**. We consider maintainers to
+be the people who contribute, review, guide, and collectively make decisions
+about the direction and scope of the project.
+
+A typical maintainer is not only someone who has made "large" contributions, but
+someone who has shown they are continuously committed to the project and its
+community. Some expected responsibilities of maintainers include (but are not
+exclusively limited to):
+
+- Displaying a high level of commitment to the project and its community, and
+being a role model for others.
+- Writing code &mdash; a lot of code, especially "glue code" or "grunt work";

```

Comment:
Do we want to say anything about documentation? Is it expected that a documentation-focused maintainer can never write a line of executed code in their time with the project, but still be an expert and guide the usability aspects of the project? Although I'd be quite happy to see that apply to first-party documentation. It doesn't make sense to me, for example, that someone who spills a lot of digital ink about jj on their own personal blog should automagically get to be a maintainer - that doesn't check the "commitment to the health of the project" box for me.